src/site/content/en/blog/coop-coep/index.md

```diff
@@ -23,27 +23,34 @@ origin_trial:
 feedback:
   - api
 ---
+{% Aside 'gotchas' %}
+**[15th Oct. 2020]**: `self.crossOriginIsolated` and Shared Array Buffer on Android
+Chrome are now available in Chrome 87. `document.domain` is now immutable when
+`self.crossOriginIsolated` returns `true`. `performance.measureMemory` in on
+origin trial from Chrome 87.
```

Comment:
```suggestion
`self.crossOriginIsolated` returns `true`. `performance.measureMemory` in on
origin trial from Chrome 87.
```
```suggestion
`self.crossOriginIsolated` returns `true`. `performance.measureMemory()` is
ending its origin trial and will be enabled by default in Chrome 88.
```

Reply:
Are we sure `performance.measureMemory` will be also shipped in Chrome 88?

Reply:
@ulan AFAIK the OT for performance.measureMemory() is ending and the API will be shipped gated behind COOP/COEP in M88, correct?
@agektmr enabled by default gated behind COOP/COEPm which is a change to the OT as the OT was not gated yet

Reply:
I can't find Intent to Ship for performance.measureMemory(). Without that I'm not sure if that's something we should say on a blog post.

Reply:
No I2S yet send. I've kicked off a thread with ulan to discuss earlier to day, will add you to that one.

When are you planning to release the blog post?

Reply:
Ideally before 15th (Thursday) given the beta announcement article goes out on that day.

Reply:
As discussed offline: From my point of view we can mention the API already in the blog post (OT ending in M87 and planned to ship in M88)

Reply:
> Ideally before 15th (Thursday) given the beta announcement article goes out on that day.

The beta announcement goes up as soon as I see the beta on the dashboards. That usually happens around 11:00 am PST.

Reply:
Please check once again: https://deploy-preview-4042--web-dev-staging.netlify.app/coop-coep/
Sqoosh's adoption is exciting and would love to add somewhere in the article but I can't find a good place. Any suggestions?

Reply:
Right, that's not easy.

What about at the end of the section 'Deploy COOP and COEP to make your website cross-origin isolated' pointing to the example?

Reply:
@lutzvahl Please check how it looks: https://github.com/GoogleChrome/web.dev/pull/4042/files#diff-5afd2bd259d9f1c14af088410be752de83c1eb1a05e55e8fae8cd0bb0d1df7e3R181

https://deploy-preview-4042--web-dev-staging.netlify.app/coop-coep/#determine-whether-isolation-succeeded-with-self.crossoriginisolated

Reply:
I would adjust the text to:
squoosh.app now uses COOP / COEP to gain access to Wasm Threads (and Shared Array Buffer) as well on Android Chrome.

Position and styling looks good!

Reply:
https://github.com/GoogleChrome/web.dev/pull/4042/commits/ad6f9fcfe95f39da5b35c2227e251e653a324459
Thank you for your review!